Ghost CMS migrator, more updates

iuravic opened the pull request:
- fixes a deprecated warning for non-explicit type
- adds docs on how to download images and other attachment files using Newspack Post Image Downloader

iuravic committed:
Fix deprecated message, use explicit nullable type

iuravic committed:
Add instructions on how to download images and files after the import

iuravic committed:
PHPCS

ronchambers commented on docs/GhostCMS.md:
```diff
@@ -91,6 +91,43 @@ The following output logs will be created:
 
+### Step 7: Download files from external host
+
+The content has now been imported, but the images are still hosted on the external host. You will need to download the images to your local WordPress installation. Recommended to use the [Newspack Post Image Downloader](https://wordpress.org/plugins/newspack-post-image-downloader/) plugin to download the images.
```
You may want to specific that it's the "in-content" images are still hosted on the external host, since the featured images and avatars were fetched already.

ronchambers replied:
See "suggested change":

* You may want to specify that it's the "in-content" images are still hosted on the external host, since the featured images and avatars were fetched already.
* Broken link.



```suggestion
The content has now been imported, but the in-content images are still hosted on the external host. You will need to download the images to your local WordPress installation. Recommended to use the [Newspack Post Image Downloader](https://github.com/Automattic/newspack-post-image-downloader) plugin to download the images.
```

ronchambers commented on docs/GhostCMS.md:
````diff
@@ -91,6 +91,43 @@ The following output logs will be created:
+wp plugin install https://github.com/Automattic/newspack-post-image-downloader/releases/latest/download/newspack-post-image-downloader.zip --activate
+```
+
+Here is the recommended workflow:
````
It's nice to have a shortened workflow here, but it also seem redundant since the steps are also in the post image downloader [readme](https://github.com/Automattic/newspack-post-image-downloader).  I don't have a strong opinion either way, but I'd recommend this:

```suggestion
Here is the recommended workflow (all available options [here](https://github.com/Automattic/newspack-post-image-downloader)):
```

ronchambers reviewed (changes requested):
Let me know what you think about the suggestions.  Thanks!

iuravic committed:
Update README

iuravic commented:
Thanks @ronchambers , I like all your suggestions. Now updated the doc in https://github.com/Automattic/newspack-migration-tools/pull/139/commits/187fc4a44c9b603208e2278d2eb93628d1f6b768 .

As for the dupe workflow for Newspack Post Image Downloader usage, it made me realize that none of the two READMEs were good enough:
- the NPID README covered all the options, but was missing the simple step-by-step practical oriented approach, and was too verbose
- this one here contained a much simpler and more practical workflow.

So I now updated the NPID's README with a new, compacted, more practical and shorter workflow, and just linked to that one from this doc, as you recommended.

ronchambers commented on docs/GhostCMS.md:
```diff
@@ -91,6 +91,12 @@ The following output logs will be created:
+
+The content has now been imported, but the images are still hosted on the external host. You will need to download the images to your local WordPress installation.
+
+Recommended to use the [Newspack Post Image Downloader](https://github.com/Automattic/newspack-post-image-downloader) plugin to download the images. Follow the [README.md](https://github.com/Automattic/newspack-post-image-downloader)) file in plugin repo for usage and best workflow.
```
```suggestion
Recommended to use the [Newspack Post Image Downloader](https://github.com/Automattic/newspack-post-image-downloader) plugin to download the images. Follow the [README.md](https://github.com/Automattic/newspack-post-image-downloader) file in plugin repo for usage and best workflow.
```

ronchambers committed:
Update docs/GhostCMS.md

Removed extra ")" character.

ronchambers reviewed (approved):
I committed and resolved one small fix to remove an extra ")" character.  Approved!
